public/game/js/PlayState/states/init.js

```diff
@@ -6,6 +6,14 @@ export function init(data) {
         right: Phaser.KeyCode.RIGHT,
         up: Phaser.KeyCode.UP
     });
+
+    this.keys.up.onDown.add(function () {
+        let didJump = this.hero.jump();
+        if (didJump) {
+            this.sfx.jump.play();
+        }
+    }, this);
+    
```

Comment:
We already have this code in handleInput.js
You can remove this

Reply:
Gotcha, should I also remove the audio files I added? I'm guessing you wanna find another sound for picking up things and jumping

Reply:
Yeah you can remove those too, they are pretty annoying and loud. We might not even need a sound to play at all tbh